.github/pull_request_template.md

```diff
@@ -20,14 +24,10 @@ Mark which items have been completed:
 
 - [ ] Refactored code ([sourcery](https://sourcery.ai/))
 - [ ] Tested code locally
+- [ ] Precommit installed and run before pushing changes
 - [ ] Added code to GitHub tests ([notebooks](workflows/test-notebooks.yml), [scripts](workflows/test-scripts.yml))
 - [ ] Updated GitHub [README](../README.md)
-
-### 🪝 Run the following to both [pre-commit](https://pre-commit.com/) hooks and JupyterLab (turn on `format on save`)
-
-- [ ] Stripped outputs from notebooks - ([nbstripout](https://pypi.org/project/nbstripout/))
-- [ ] Sorted imports - ([isort](https://pycqa.github.io/isort/))
-- [ ] Formatted code - ([back](https://github.com/psf/black))
+- [ ] Updated the [projects overview page](https://www.notion.so/neptuneai/0e1692ba6547450eb873bc23982dd898?v=0dfa115ffe754074806ec97638878736)
```

Comment:
Can we make this invisible to the public?

Reply:
no unfortunately. The page is private, but I can remove the link if that's a security concern

Reply:
I think it might be

Reply:
Just general direction (name of the page) is enough